sqlglot/dialects/bigquery.py

```diff
@@ -330,7 +330,8 @@ class Tokenizer(tokens.Tokenizer):
             "RECORD": TokenType.STRUCT,
             "TIMESTAMP": TokenType.TIMESTAMPTZ,
         }
-        KEYWORDS.pop("DIV")
+        KEYWORDS.pop("DIV", None)
```

Comment:
why did you have to add None here?

Reply:
I started by doing that for `"VALUES"` with the reasoning that it would protect by throwing a key error in the future if there's no keyword to pop for whatever reason, and then I updated that to apply the same logic. Should I revert?

Reply:
yea, i don't understand why that's necessary,  get rid of the None, it's unnecessary